.env

```diff
@@ -0,0 +1,6 @@
+PGPASSWORD=password
```

Comment:
You should _never_ commit this file. This is where you store all the sensitive connection info, its good practice to add this to the `.gitignore` file to prevent adding this accidentally